modules/ztp-install-factory-pipeline-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="factory-install-workflow_{context}"]
+= The factory install workflow
+
+The factory install pipeline builds out your factory environment in readiness for the edge cluster to be shipped off site.
+
+The following diagram provides a high level overview of the pipelines used to prepare the edge clusters:
+
+.Hub and edge pipelines
+
+image::225_OpenShift_Installing_Clusters_0422_pipeline.png[Factory install workflow]
+
+* **Hub deployment**: This first part deploys the hub cluster configuration. The assumption being {product-title} and optionally OpenShift Data Foundation is installed with persistent volumes created with supporting DHCP and DNS configuration.
+* **Edge deployment**: This second part deploys relocatable edge clusters on the preferred hardware in parallel. When the deployment completes, the edge cluster (with hardware) is shipped to the end customer. The end customer runs some on site configuration steps and then has a fully operational {product-title} cluster.
```

Comment:
instead of
"the edge cluster (with hardware)"
The hardware where the edge cluster is installed